README.md

```diff
@@ -2,9 +2,225 @@
+the json response schema.
+
+### Getting captions output
+
+Another way to retrieve your file is captions output. We support both .srt and .vtt outputs. 
+See below for an example showing how you can get captions as a readable stream. If your job 
+was submitted with multiple speaker channels you are required to provide the id of the channel 
+you would like captioned.
+
+```
+InputStream inputStream = apiClient.getCaptions(revAiJob.getJobID(), RevAiCaptionType.SRT);
+
+// with speaker channels
+int channelId = 1;
+InputStream inputStream = apiClient.getCaptions(revAiJob.getJobID(), RevAiCaptionType.VTT, channelId);
+```
+
+## Streaming Audio
+
+In order to stream audio, you will need to setup a streaming client and the content type 
+for the audio you will be sending.
+
+```
+StreamContentType streamContentType = new StreamContentType();
+    streamContentType.setContentType("audio/x-raw");
+    streamContentType.setLayout("interleaved");
+    streamContentType.setFormat("S16LE");
+    streamContentType.setRate(16000);
+    streamContentType.setChannels(1);
+
+StreamingClient streamingClient = new StreamingClient("Your Access Token");
+```
+
+You will need to create Listener that implements the RevAiWebSocketListener in order 
+to handle WebSocket events.
+
+```java
+public class Listener implements RevAiWebSocketListener {
+
+    @Override
+    public void onConnected(ConnectedMessage message) {
+        System.out.println("On Connected: " + connectMessage);
+    }
+
+    @Override
+    public void onHypothesis(Hypothesis hypothesis) {
+        System.out.println("On Hypothesis: " + hypothesis);
+    }
+
+    @Override
+    public void onError(Throwable t, Response response) {
+        System.out.println("On Error: " + response.toString());
+    }
+
+    @Override
+    public void onClose(int code, String reason) {
+        System.out.println("On Close: [" + code + "] " + reason);
+    }
+
+    @Override
+    public void onOpen(Response response) {
+        System.out.println("On Open: " + response.toString());
+    }
+}
+```
+
+Now you will be able to connect and start the streaming session by calling the `streamingClient.connect()` method and passing in the Listener! You can supply an optional `SessionConfig` object to the function in order to provide additional information for that session, such as metadata or enabled the profanity filter to be used.
+
+```
+Listener listener = new Listener();
+
+SessionConfig sessionConfig = new SessionConfig();
+sessionConfig.setMetaData("My first job");
+sessionConfig.setFilterProfanity(true);
+
+streamingClient.connect(clientListener, streamContentType, sessionConfig);
+```
+
+You can stream data over the WebSocket in the form of a `ByteString` using the `streamingClient.sendAudioData()` method.
+
+```
+streamingClient.sendAudioData(ByteString);
+```
+
+The streaming connection will close when you call the method `streamingClient.close()` or if you go 15 seconds without sending any audio data.
```

Comment:
It would be good to provide guidance on how a StreamingClient instance should be reused (or not) across multiple streaming sessions (both simultaneous and not).